CHANGELOG.md

```diff
@@ -9,7 +9,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Enhancements
 
--
+- Added the ability to set `PriorityClass` through ScanSetting. All the scanner pods
+  will be launched using the `PriorityClass` if set.
```

Comment:
We could also say this is an optional feature and that scans will continue to work.